app/src/main/res/layout/activity_artist.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerARTIST"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
it's not better to add match parent and match parent and remove unncesessary constraints to parent.